TypeCobol/Compiler/CodeElements/Entry/DataDescriptionEntry.cs

```diff
@@ -52,8 +53,7 @@ public abstract class DataDefinitionEntry: NamedCodeElement {
 		/// </summary>
 		[CanBeNull]
 		public IntegerValue LevelNumber { get; set; }
-
-		/// <summary>
+        /// <summary>
```

Comment:
unwanted change

Reply:
@laabidiHend Can you check this comment.